app/controllers/green_lanes/results_controller.rb

```diff
@@ -29,6 +30,12 @@ def results_params
       )
     end
 
+    def cas_without_exemptions
```

Comment:
I guess cas is category assessments :thinking:

Reply:
Yeah sorry i can change that if needed